pkg/system/phase4_configuring.go

```diff
@@ -1281,12 +1345,22 @@ func (r *Reconciler) ReconcileDefaultBucketClass() error {
 		return nil
 	}
 
-	r.DefaultBucketClass.Spec.PlacementPolicy = &nbv1.PlacementPolicy{
-		Tiers: []nbv1.Tier{{
-			BackingStores: []nbv1.BackingStoreName{
-				r.DefaultBackingStore.Name,
+	if util.KubeCheck(r.DefaultBackingStore) {
+		r.DefaultBucketClass.Spec.PlacementPolicy = &nbv1.PlacementPolicy{
+			Tiers: []nbv1.Tier{{
+				BackingStores: []nbv1.BackingStoreName{
+					r.DefaultBackingStore.Name,
+				},
+			}},
+		}
+	}
+	if util.KubeCheck(r.DefaultNamespaceStore) {
```

Comment:
add else - no need to check both (correct me if I'm wrong...)

Reply:
Done.